tests/acceptance/expectUserAndAdminLoginCest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ Cest is a class based format. Cest format is able to support multiple 
+ tests per file and easy reuse of code by adding additional private functions.
+ Cest combines scenario-driven test approach with OOP design. 
+ */
+class expectUserAndAdminLoginCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        //runs before each test
+        $I->amOnPage('/');
+    }
+    public function _after(AcceptanceTester $I)
+    {
+        //runs after each test
+    }
+    // tests
+    
+    public function expectsUserToLogin(AcceptanceTester $I) {
+        $I->wantTo('log in as a user');
+        $I->loginAs('chappress', 'password');
```

Comment:
I tested this with a non-admin user I created locally, but I wasn't sure how to update `chappress_dev_dump`the correct way so that it would generate the database for other devs with the non-admin user I created.